pkg/kudoctl/cmd/get/get.go

```diff
@@ -3,60 +3,144 @@ package get
 import (
 	"errors"
 	"fmt"
-	"log"
+	"io"
 
 	"github.com/xlab/treeprint"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
 
-	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/cmd/output"
 	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
 )
 
+type CmdOpts struct {
+	Out    io.Writer
+	Output output.Type
+
+	Namespace string
+	Client    *kudo.Client
```

Comment:
Yeah... but `Out io.Writer` isn't a parameter either. And it's done this way on all Cmds. I'd rather keep it this way and have it the same way on all Cmds instead of doing it differently here.